payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/LpmSerializerTest.kt

```diff
@@ -67,21 +67,22 @@ class LpmSerializerTest {
         val result = lpmSerializer.deserializeList(serializedString)
 
         val dropdownSpec = result.first { it.type == "eps" }
-            .fields[1] as DropdownSpec
+            .fields[3] as DropdownSpec
```

Comment:
This needed to be changed because I updated the lpm specs to include placeholders, they match what's currently in the server specs, I did the same for iOS